DuckDuckGo/ConfigurationDebugViewController.swift

```diff
@@ -153,6 +162,18 @@ class ConfigurationDebugViewController: UITableViewController {
             case .resetLastRefreshDate:
                 lastConfigurationRefreshDate = Date.distantPast
                 tableView.reloadData()
+            case .copyConfigLocationPath:
+                let location = FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: ContentBlockerStoreConstants.groupName)
+                UIPasteboard.general.string = location?.path ?? ""
```

Comment:
It wasn't obvious to me that selecting this cell copied the thing to the paste board, can we change the text slightly to indicate that?

Reply:
Sure! What do you think about:
- Copy Location of Config Files to Pasteboard
- Send Location of Config Files to Pasteboard

or do you have other suggestion?

Reply:
I like the first one!